export_test.go

```diff
@@ -35,48 +37,116 @@ const (
+	ComputeDynamicPolicy                     = computeDynamicPolicy
+	ComputeSecureBootPolicyDigests           = computeSecureBootPolicyDigests
+	ComputeStaticPolicy                      = computeStaticPolicy
+	CreatePinNVIndex                         = createPinNVIndex
+	CreatePublicAreaForRSASigningKey         = createPublicAreaForRSASigningKey
+	DecodeSecureBootDb                       = decodeSecureBootDb
+	DecodeWinCertificate                     = decodeWinCertificate
+	EkTemplate                               = ekTemplate
+	EFICertTypePkcs7Guid                     = efiCertTypePkcs7Guid
+	EFICertX509Guid                          = efiCertX509Guid
+	EnsureLockNVIndex                        = ensureLockNVIndex
+	ExecutePolicySession                     = executePolicySession
+	IdentifyInitialOSLaunchVerificationEvent = identifyInitialOSLaunchVerificationEvent
+	IncrementDynamicPolicyCounter            = incrementDynamicPolicyCounter
+	LockAccessToSealedKeysUntilTPMReset      = lockAccessToSealedKeysUntilTPMReset
+	LockNVIndexAttrs                         = lockNVIndexAttrs
+	MakeDefaultEKTemplate                    = makeDefaultEKTemplate
+	OidExtensionSubjectAltName               = oidExtensionSubjectAltName
+	OidTcgAttributeTpmManufacturer           = oidTcgAttributeTpmManufacturer
+	OidTcgAttributeTpmModel                  = oidTcgAttributeTpmModel
+	OidTcgAttributeTpmVersion                = oidTcgAttributeTpmVersion
+	OidTcgKpEkCertificate                    = oidTcgKpEkCertificate
+	PerformPinChange                         = performPinChange
+	ReadAndValidateLockNVIndexPublic         = readAndValidateLockNVIndexPublic
+	ReadDynamicPolicyCounter                 = readDynamicPolicyCounter
+	ReadShimVendorCert                       = readShimVendorCert
+	SrkTemplate                              = srkTemplate
+	WinCertTypePKCSSignedData                = winCertTypePKCSSignedData
+	WinCertTypeEfiGuid                       = winCertTypeEfiGuid
 )
 
-var ComputeDynamicPolicy = computeDynamicPolicy
-var ComputeStaticPolicy = computeStaticPolicy
-var CreatePinNVIndex = createPinNVIndex
-var CreatePublicAreaForRSASigningKey = createPublicAreaForRSASigningKey
-var EnsureLockNVIndex = ensureLockNVIndex
-var ExecutePolicySession = executePolicySession
-var IncrementDynamicPolicyCounter = incrementDynamicPolicyCounter
-var LockAccessToSealedKeysUntilTPMReset = lockAccessToSealedKeysUntilTPMReset
-var PerformPinChange = performPinChange
-var ReadAndValidateLockNVIndexPublic = readAndValidateLockNVIndexPublic
-var ReadDynamicPolicyCounter = readDynamicPolicyCounter
-
+// Alias some unexported types for testing. These are required in order to pass these between functions in tests, or to access
+// unexported members of some unexported types.
 type DynamicPolicyData dynamicPolicyData
 
-type MockPolicyPCRParam struct {
-	PCR     int
-	Alg     tpm2.HashAlgorithmId
-	Digests tpm2.DigestList
+type EFISignatureData efiSignatureData
```

Comment:
a type alias woudl be:

type EFISignatureData = efiSignatureData

Reply:
Aha, I didn't know there was a difference but I've just had a look at the documentation. I'll fix that and others in another PR.